gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -136,25 +136,16 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
+
     parser.add_argument(
         '--sharding_config_path',
-        default=('gcp_variant_transforms/data/sharding_configs/'
-                 'homo_sapiens_default.yaml'),
-        help=('File containing list of output tables, their name suffixes, and '
-              'approximate number of total base pairs which is used to conduct '
-              'BigQuery integer range partitioning. Default value is set to a '
-              'file which is optimized for the human genome. It results in one '
-              'table per chromosome (overall 25 BigQuery tables). For more '
-              'information visit gcp-variant-trannsforms/docs/sharding.md'))
-    parser.add_argument(
-        '--generate_sample_info_table',
-        type='bool', default=False, nargs='?', const=True,
-        help=('If set to True, a sample info table with the name '
-              'output_table_ + {} will be created. This table contains a '
-              'unique sample_id for each sample read from the VCF file. This '
-              'sample_id can be used to distinguish between sample names. '
-              '[EXPERIMENTAL]'
-             ).format(sample_info_table_schema_generator.TABLE_SUFFIX))
+        default='gcp_variant_transforms/data/sharding_configs/'
+                'homo_sapiens_default.yaml',
+        help=('File containing list of shards and output table names. You '
+              'can use provided default sharding_config file to split output '
+              'by chromosome (one table per chromosome) which is located at: '
+              'gcp_variant_transforms/data/sharding_configs/'
+              'homo_sapiens_default.yaml'))
```

Comment:
Any specific reason you decided to update the doc we agreed on in the previous PR?

Reply:
Nice catch! Sync error, applied your changes myself (that's how I caught the other issues I guess) - reverted.